### PR TITLE
Refreshed english translations

### PR DESCRIPTION
**Pull request by teolemon**
The en.po had fallen behind common.pot because no export is done, unlike for en_au.po

### CODE REVIEW

File: po/common/en.po [hunk 1]
```diff
@@ -186,16 +192,15 @@ msgstr "Barcode number:"
...
-msgstr "<a href=\"https://www.helloasso.com/associations/open-food-facts/formulaires/1/widget/en\"><img src=\"https://static.openfoodfacts.org/images/svg/donate-icon.svg\" alt=\"Donate to Open Food Facts\" /></a><p><<site_name>> is made by a non-profit association, independent from the industry. It is made for all, by all, and it is funded by all. You can support our work by <a href=\"https://www.helloasso.com/associations/open-food-facts/formulaires/1/widget/en\">donating to Open Food Facts</a> and also by <a href=\"https://www.lilo.org/fr/open-food-facts/?utm_source=open-food-facts\">using the Lilo search engine</a>.<br/><b>Thank you!</b></p>"
-
+msgstr ""
 
 msgctxt "bottom_title"
-msgid "Donate to support our work"
-msgstr "Donate to support our work"
+msgid "Embrace the mission"
```
Comment (stephanegigandet) [inline po/common/en.po]: This one needs to be changed back.

File: po/common/en.po [hunk 2]
```diff
@@ -186,16 +192,15 @@ msgstr "Barcode number:"
...
-msgstr "<a href=\"https://www.helloasso.com/associations/open-food-facts/formulaires/1/widget/en\"><img src=\"https://static.openfoodfacts.org/images/svg/donate-icon.svg\" alt=\"Donate to Open Food Facts\" /></a><p><<site_name>> is made by a non-profit association, independent from the industry. It is made for all, by all, and it is funded by all. You can support our work by <a href=\"https://www.helloasso.com/associations/open-food-facts/formulaires/1/widget/en\">donating to Open Food Facts</a> and also by <a href=\"https://www.lilo.org/fr/open-food-facts/?utm_source=open-food-facts\">using the Lilo search engine</a>.<br/><b>Thank you!</b></p>"
-
+msgstr ""
```
Comment (stephanegigandet) [inline po/common/en.po]: Needs to be added back.

File: po/common/en.po [hunk 3]
```diff
@@ -331,19 +336,19 @@ msgstr "Please correct the following errors:"
 
 msgctxt "correctors_p"
 msgid "correctors"
-msgstr "correctors"
+msgstr "corrections"
```
Comment (stephanegigandet) [inline po/common/en.po]: bad

File: po/common/en.po [hunk 4]
```diff
@@ -331,19 +336,19 @@ msgstr "Please correct the following errors:"
...
 
 msgctxt "correctors_s"
 msgid "corrector"
-msgstr "corrector"
+msgstr "correction"
 
 msgctxt "countries"
 msgid "Countries where sold"
-msgstr "Countries where sold"
+msgstr "Country"
```
Comment (stephanegigandet) [inline po/common/en.po]: bad

File: po/common/en.po (hunk 2 above)
Comment (teolemon) [reply po/common/en.po]: Can we break down those awfully long, French specific strings into parts ?

File: po/common/en.po (hunk 4 above)
Comment (teolemon) [reply po/common/en.po]: Please make a PR against common.pot

File: po/common/en.po (hunk 3 above)
Comment (teolemon) [reply po/common/en.po]: Please make a PR against common.pot

File: po/common/en.po (hunk 2 above)
Comment (teolemon) [reply po/common/en.po]: Please make a PR against common.pot

File: po/common/en.po (hunk 1 above)
Comment (teolemon) [reply po/common/en.po]: I've made a PR against common.pot

**Comment by stephanegigandet** [comment]
There are many missing translations, and a few bad ones.

File: po/common/en.po (hunk 1 above)
Comment (teolemon) [reply po/common/en.po]: https://github.com/openfoodfacts/openfoodfacts-server/pull/2337

**Comment by teolemon** [comment]
The missing translations: This is English, so it doesn't matter much at the end of the day. 
Bad ones: separate PR

File: po/common/en.po (hunk 1 above)
Comment (teolemon) [inline po/common/en.po]:
```suggestion
msgid "Donate to support our work"
```

File: po/common/en.po [hunk 5]
```diff
@@ -186,16 +192,15 @@ msgstr "Barcode number:"
...
-msgstr "<a href=\"https://www.helloasso.com/associations/open-food-facts/formulaires/1/widget/en\"><img src=\"https://static.openfoodfacts.org/images/svg/donate-icon.svg\" alt=\"Donate to Open Food Facts\" /></a><p><<site_name>> is made by a non-profit association, independent from the industry. It is made for all, by all, and it is funded by all. You can support our work by <a href=\"https://www.helloasso.com/associations/open-food-facts/formulaires/1/widget/en\">donating to Open Food Facts</a> and also by <a href=\"https://www.lilo.org/fr/open-food-facts/?utm_source=open-food-facts\">using the Lilo search engine</a>.<br/><b>Thank you!</b></p>"
-
+msgstr ""
 
 msgctxt "bottom_title"
-msgid "Donate to support our work"
-msgstr "Donate to support our work"
+msgid "Embrace the mission"
+msgstr "Embrace the mission"
```
Comment (teolemon) [inline po/common/en.po]:
```suggestion
msgstr "Donate to support our work"
```

File: po/common/en.po (hunk 3 above)
Comment (teolemon) [inline po/common/en.po]:
```suggestion
msgstr "correctors"
```

File: po/common/en.po [hunk 6]
```diff
@@ -331,19 +336,19 @@ msgstr "Please correct the following errors:"
 
 msgctxt "correctors_p"
 msgid "correctors"
-msgstr "correctors"
+msgstr "corrections"
 
 msgctxt "correctors_s"
 msgid "corrector"
-msgstr "corrector"
+msgstr "correction"
```
Comment (teolemon) [inline po/common/en.po]:
```suggestion
msgstr "corrector"
```

File: po/common/en.po (hunk 4 above)
Comment (teolemon) [inline po/common/en.po]:
```suggestion
msgstr "Countries where sold"
```

File: po/common/en.po [hunk 7]
```diff
@@ -547,7 +545,7 @@ msgstr "Incorrect user name or password. <a href=\"/cgi/reset_password.pl\">Forg
 
 msgctxt "error_database"
 msgid "An error occurred while reading the data, try to refresh the page."
-msgstr "An error occurred while reading the data, try to refresh the page."
+msgstr ""
```
Comment (teolemon) [inline po/common/en.po]:
```suggestion
msgstr "An error occurred while reading the data, try to refresh the page."
```

File: po/common/en.po [hunk 8]
```diff
@@ -808,7 +801,7 @@ msgstr "See you soon!"
 
 msgctxt "graph_count"
 msgid "%d products match the search criteria, of which %i products have defined values for the graph's axis."
-msgstr "%d products match the search criteria, of which %i products have defined values for the graph's axis."
+msgstr ""
```
Comment (teolemon) [inline po/common/en.po]:
```suggestion
msgstr "%d products match the search criteria, of which %i products have defined values for the graph's axis."
```

File: po/common/en.po [hunk 9]
```diff
@@ -895,28 +866,28 @@ msgid "You must enter the characters of the barcode or send a product image when
...
-msgid "Full size";
-msgstr "Full size"
+msgid "Full size"
+msgstr ""
 
 msgctxt "image_attribution_creativecommons"
-msgid "This file was uploaded to product %s and is licensed under the %s license.";
-msgstr "This file was uploaded to product %s and is licensed under the %s license."
+msgid "This file was uploaded to product %s and is licensed under the %s license."
+msgstr ""
```
Comment (teolemon) [inline po/common/en.po]:
```suggestion
msgstr "This file was uploaded to product %s and is licensed under the %s license."
```

File: po/common/en.po [hunk 10]
```diff
@@ -895,28 +866,28 @@ msgid "You must enter the characters of the barcode or send a product image when
...
 msgctxt "image_attribution_photographer"
-msgid "Attribution: Photo by %s per %s";
-msgstr "Attribution: Photo by %s per %s"
+msgid "Attribution: Photo by %s per %s"
+msgstr ""
 
 msgctxt "image_attribution_photographer_editor"
 msgid "Attribution: Photo by %s with additional modifications by %s per %s"
-msgstr "Attribution: Photo by %s with additional modifications by %s per %s"
+msgstr ""
```
Comment (teolemon) [inline po/common/en.po]:
```suggestion
msgstr "Attribution: Photo by %s with additional modifications by %s per %s"
```

File: po/common/en.po [hunk 11]
```diff
@@ -895,28 +866,28 @@ msgid "You must enter the characters of the barcode or send a product image when
...
 
 msgctxt "image_original_link_text"
 msgid "(Original Image)"
-msgstr "(Original Image)"
+msgstr ""
```
Comment (teolemon) [inline po/common/en.po]:
```suggestion
msgstr "(Original Image)"
```

**Comment by teolemon** [comment]
I used the en-au file to create this file. It doesn't matter much actually.

**Comment by teolemon** [comment]
``` 
Testing po/common/en.po .po fileError: empty msgstr string for msgctxt add_user_email_body - msgid 
Error: empty msgstr string for msgctxt licence_accept - msgid 
Error: empty msgstr string for msgctxt login_register_content - msgid 
Error: empty msgstr string for msgctxt login_to_add_products - msgid 
Error: empty msgstr string for msgctxt nutrition_data_table_note - msgid 
Error: empty msgstr string for msgctxt openfoodhunt_points - msgid 
Error: empty msgstr string for msgctxt reset_password_email_body - msgid 
Error: empty msgstr string for msgctxt search_graph_instructions - msgid 
Error: empty msgstr string for msgctxt warning_3rd_party_content - msgid 
```

File: po/common/en.po [hunk 12]
```diff
@@ -938,10 +927,6 @@ msgctxt "ingredients_alt"
 msgid "Ingredients"
 msgstr "Ingredients"
 
-msgctxt "ingredients_analysis"
```
Comment (teolemon) [inline po/common/en.po]: @stephanegigandet ?

File: po/common/en.po [hunk 13]
```diff
@@ -1518,19 +1504,19 @@ msgstr "nutrient level"
 
 msgctxt "nutriments_p"
 msgid "nutriments"
-msgstr "nutriments"
+msgstr "nutrients"
```
Comment (teolemon) [inline po/common/en.po]: nutrient or nutriments @stephanegigandet ?

**Pull request merged**